detect/tasks/sma.go

```diff
@@ -34,8 +36,9 @@ func SMAHandlerFactory(conf map[string]interface{}) (TaskHandler, error) {
 
 type SMAHandler struct {
 	mux      sync.Mutex
-	listener *sma.Listener
+	handled  bool
```

Comment:
noch schöner wäre wohl das mit sync.Once zu machen, aber nicht kriegsentscheidend

Reply:
ist aber auch nicht ganz perfekt da sync.once ja eigentlich dafür gedacht einen Funktionsaufruf zu kapseln. 
Hier ist das ja die Methode die nur einmal aufgerufen werden soll (da sieht das mit sync.once glaube ich auch nicht schön aus)

Reply:
yep- dann müssten die Innereien in eine Methode. Ich hänge nicht dran ;)

Reply:
Ich glaube das lasse ich erstmal so.
Vielleicht macht es sinn den detector für so etwas generell zu befähigen (braucht man bestimmt nicht nur für SMA)